pkg/utils/metrics/metrics.go

```diff
@@ -63,3 +80,21 @@ func NewLabelTimer(subsystem, name, help string, labels ...string) *prometheus.H
 			ConstLabels: nil,
 		}, labels)
 }
+
+// GetCustomPrometheusRegistry create a custom resigtry of Prometheus.
+func GetCustomPrometheusRegistry() *prometheus.Registry {
```

Comment:
I think we should use `GetPrometheusRegistry` here. The `Custom` is for the prometheus package, not for pouch. Right?